### PR TITLE
Update integrations docs to use to new doc template

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

File: logstash/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# ![](https://github.com/signalfx/integrations/blob/master/logstash/img/integration_logstash.png) Logstash
```
Comment (alstory) [inline logstash/README.md]: this link is 404

File: logstash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+browse_categories:
+- Analytics
+code: https://github.com/signalfx/signalfx-agent
+data_signature: "plugin:\"logstash\" OR plugin:\"logstash-tcp\""
+description: Monitor Logstash and comsume metrized events from Logstash TCP output plugin using the SignalFx Smart agent.
```
Comment (alstory) [inline logstash/meta.yaml]: Is: comsume
change to: consume

**Comment by alstory** [comment]
just a typo

File: logstash/meta.yaml (hunk 2 above)
Comment (asuresh4) [reply logstash/meta.yaml]: updated

File: logstash/README.md (hunk 1 above)
Comment (asuresh4) [reply logstash/README.md]: this is because the image points to master. link will be fixed once changes are in master

**Comment by asuresh4** [comment]
@alstory - I've updated the pr with changes from the new template. It would be great if you could take look to see if there's regression.

**Pull request closed without merging**